code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/service/impl/RancherImageToComposeFormatter.java

```diff
@@ -0,0 +1,19 @@
+package io.cattle.platform.servicediscovery.service.impl;
+
+import io.cattle.platform.servicediscovery.resource.ServiceDiscoveryConfigItem;
+
+public class RancherImageToComposeFormatter implements RancherConfigToComposeFormatter {
+    private static final String IMAGE_PREFIX = "docker:";
```

Comment:
At the moment, we don't have tests for docker-compose.yml export. I think it should be added to the validation set though - export the config, and try to run it with docker-compose. Testing it with the data generated on simulator setup, would break anyway.